packages/itwin/tree-widget/src/components/trees/VisibilityTreeBase.scss

```diff
@@ -3,109 +3,77 @@
-    &.with-checkbox {
-      >.contents {
-        padding-left: calc(var(--iui-size-3xs) / 2);
+  .tree-widget-tree-nodes-list {
+    .core-tree-node {
+      &.with-checkbox {
+        >.contents {
+          padding-left: calc(var(--iui-size-3xs) / 2);
+        }
       }
-    }
 
-    &.disable-expander {
-      >.contents {
-        >.core-tree-expansionToggle {
-          display: none;
+      &.disable-expander {
+        >.contents {
+          >.core-tree-expansionToggle {
+            display: none;
+          }
         }
       }
-    }
 
-    &.is-selected {
-      >.contents {
-        >.visibility-tree-checkbox-container {
-          background-color: var(--iui-color-background-accent-muted)
+      &.is-selected {
+        >.contents {
+          >.visibility-tree-checkbox-container {
+            background-color: var(--iui-color-background-accent-muted)
+          }
         }
       }
-    }
-  }
 
-  .contents {
-    &:hover {
-      > .visibility-tree-checkbox-container {
-        background-color: var(--iui-color-background-hover);
+      .contents {
+        &:hover {
+          .visibility-tree-checkbox-container {
+            background-color: var(--iui-color-background-hover);
+          }
+        }
       }
-    }
 
-    > .visibility-tree-checkbox-container {
-      height: var(--iui-size-l);
-      width: var(--iui-size-l);
-      position: sticky;
-      order: -1;
-      left: 0;
-      background-color: var(--iui-color-background);
-      z-index: 1;
-      display: flex;
-      align-items: center;
-      justify-content: center;
+      .visibility-tree-checkbox-container {
+        height: var(--iui-size-l);
+        width: var(--iui-size-l);
+        position: sticky;
+        order: -1;
+        left: 0;
+        background-color: var(--iui-color-background);
+        z-index: 1;
+        display: flex;
+        align-items: center;
+        justify-content: center;
 
-      .visibility-tree-checkbox {
-        &:disabled {
-          background-color: transparent;
-        }
+        .visibility-tree-checkbox {
+          &:disabled {
+            background-color: transparent;
+          }
 
-        &:hover {
-          background-color: transparent;
+          &:hover {
+            background-color: transparent;
+          }
         }
       }
     }
-  }
-
-  .filteredTreeOverlay {
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    z-index: 9999;
-    background-color: hsl(var(--iui-color-background) / var(--iui-opacity-2));
-  }
 
-  .components-tree-errormessage {
-    position: absolute;
-    top: 40%;
-    left: 50%;
-    width: 75%;
-    text-align: center;
-    transform: translate(-50%, -50%);
-    font-style: italic;
-    color: var(--iui-color-text-muted);
+    &.explode {
+      .core-tree-node {
+        .visibility-tree-checkbox-container {
+          height: $exploded-node-height;
+          width: calc($exploded-node-height * 0.75);
```

Comment:
why not 100%?

Reply:
Making checkbox width `$exploded-node-height` makes eye icon to take all that space 42x42. With current implementation checkbox is a little bit narrower but icon looks better.

Reply:
But the touch area of the checkbox is not 42x42 then, right? I manually changed the style and it seems the eye icon looks fine:
![image](https://github.com/iTwin/viewer-components-react/assets/35135765/b45ed803-f3c4-4eba-bc05-b6b0510303fa)

I guess I'm missing something?

Reply:
Should be ok now.